packages/components/src/Combobox/components/ComboboxContent/ComboboxContent.tsx

```diff
@@ -81,7 +85,17 @@ export function ComboboxContent(props: ComboboxContentProps): JSX.Element {
     </div>
   );
 
+  if (parentNodeId) {
```

Comment:
This is the big change from the initial PR. Floating UI breaks in JO so this only uses the floating UI tree logic if it is inside another Floating UI element (like modal). If not it will use the default portal behaviour